README.md

```diff
@@ -19,11 +19,15 @@ On Linux, you need to install the Qt 5 package(s) for your respective distributi
 
 ## How to Build
 ### Microsoft Windows
-> *You will need Microsoft Visual Studio 2015 or 2017 with Visual C++ installed. Previous versions may work but are untested. The solution works best with VS2017, but with a small adjustment, it is possible to have VS2015 work.*
+#### Requirements
+- Visual Studio 2017
```

Comment:
It actually works on vs2015 too and it was tested too, just indicate it, otherwise LGTM.